build.gradle.kts

```diff
@@ -22,6 +22,7 @@ buildscript {
         classpath(Config.BuildPlugins.androidGradle)
         classpath(kotlin(Config.BuildPlugins.kotlinGradlePlugin, version = Config.kotlinVersion))
         classpath(Config.QualityPlugins.errorpronePlugin)
+        classpath("io.sentry:sentry-android-gradle-plugin:1.7.28")
```

Comment:
Can this be only for the sample instead?

Reply:
@bruno-garcia this is only for the sample, classpath means available, who wants to add the plugin.

I kept it commented for now and let's discuss it later on